content/about/resources.en.md

```diff
@@ -8,7 +8,7 @@ category: about
 
 ## Logo Downloads
 
-Please review the [trademark policy](/trademark) for information about permissible use of Node.js® logos and marks.
+Please review the [trademark policy](https://openjsf.org/wp-content/uploads/sites/84/2021/01/OpenJS-Foundation-Trademark-Policy-2021-01-12.docx.pdf) for information about permissible use of Node.js® logos and marks.
```

Comment:
You don't need to put this link, as this one might change over time. I would recommend replacing that link for `https://trademark-policy.openjsf.org/`

Reply:
OK